lib/core/GraphicsFactory.js

```diff
@@ -192,6 +192,11 @@ GraphicsFactory.prototype.updateContainments = function(elements) {
     forEach(children.slice().reverse(), function(child) {
       var childGfx = elementRegistry.getGraphics(child);
```

Comment:
I struggle to understand in which situation this can become an issue, could you elaborate? And would you be able to model this as a test case in diagram-js?

From what I understand: 

* `GraphicsFactory#updateContainment` is being called after any change operation happened ([ref](https://github.com/bpmn-io/diagram-js/blob/371877b77a548525d58ac12b56143eec3eaf17b4/lib/features/change-support/ChangeSupport.js#L57))
* At this stage, looking at the model, parents for updated children are collected
* Then the children inside these parents are re-ordered
* _BUT_ some of the children are actually _NOT_ part of the render tree anymore?

The _BUT_ is the big issue for me. How can children exist, but not actually be rendered? Where don't we properly update the model _before_ we render?